packages/SwingSet/misc-tools/monitor-slog-block-time.py

```diff
@@ -3,6 +3,9 @@
 from collections import defaultdict
 import subprocess
 
+print("writing to monitor.out")
+mf = open("monitor.out", "a", buffering=1)
+
 # run like this:
 #   tail -n 10000 -F chain.slog | python3 monitor-slog-block-time.py [keys.txt]
 # produces output like:
```

Comment:
This file seems to have significant overlap with block-times.py; can they be combined?

Reply:
I'll look into it, but `monitor-slog-block-time.py` is the main tool I use for real-time monitoring of my follower node (human output),  whereas `block-times.py` is for postmortem analysis and CSV-driven graphs.